src/w3d/renderer/render2dsentence.cpp

```diff
@@ -771,6 +771,10 @@ void Render2DSentenceClass::Build_Sentence_Centered(const unichar_t *text, int *
                     break;
                 }
 
+                if (test_ch == U_CHAR('\n')) {
```

Comment:
This looks like it can be omitted because of the test above. It's odd it was compiled in though.

Reply:
Fixed.